src/ast/types/blocks-and-statements.ts

```diff
@@ -46,16 +46,26 @@ export interface EnhancedForStatement {
   kind: "EnhancedForStatement";
 }
 
-export type StatementWithoutTrailingSubstatement = Block | ExpressionStatement | DoStatement;
+export type StatementWithoutTrailingSubstatement = Block | ExpressionStatement | DoStatement | ReturnStatement;
 
-export type ExpressionStatement = MethodInvocation | Assignment;
+export interface ExpressionStatement {
+  kind: "ExpressionStatement";
+  stmtExp: StatementExpression
+}
+
+export type StatementExpression = MethodInvocation | Assignment;
 
 export interface MethodInvocation {
   kind: "MethodInvocation";
-  identifier: Identifier
+  identifier: MethodName;
   argumentList: ArgumentList;
 }
 
+export interface MethodName {
+  kind: "MethodName";
+  name: string;
+}
+
```

Comment:
I would prefer using `Identifier` directly because it is basically just a string. But if turning it into a node has advantages to the ECE, I am also okay.

Reply:
Same thing, ECE requires `MethodName` to be a `Node`.

Reply:
Reverted `MethodName` back to `Identifier` in [PR#20](https://github.com/source-academy/java-slang/pull/20).